compiler/src/main/kotlin/io/spine/protodata/backend/MessageCompilerEvents.kt

```diff
@@ -24,17 +24,23 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.protodata.event
+package io.spine.protodata.backend
 
 import com.google.protobuf.Descriptors
-import com.google.protobuf.Empty
 import io.spine.base.EventMessage
-import io.spine.protodata.Documentation
-import io.spine.protodata.Field
 import io.spine.protodata.File
-import io.spine.protodata.MessageType
 import io.spine.protodata.OneofGroup
 import io.spine.protodata.TypeName
+import io.spine.protodata.event.FieldEntered
```

Comment:
@dmdashenkov, we do need `event` package so that Javadocs, KDocs, and source code JARs are easier to understand. Please do have events coming under the `event` package.